src/RA_Dlg_MemBookmark.h

```diff
@@ -42,8 +42,16 @@ class MemBookmark
 class Dlg_MemBookmark
 {
 public:
-    //Dlg_MemBookmark();
-    //~Dlg_MemBookmark();
+    enum class SubItems
+    {
+        Desc,
+        Address,
+        Value,
+        Previous,
+        Changes
+    };
```

Comment:
this enum isn't exposed by the class. recommend keeping it private.

Reply:
sure